### DISCUSSION TITLE
AWX Connection

### DISCUSSION THREAD

**Discussion by mlesierra** [ansibleguy76/ansibleforms#211] [Q&A] [answered]
Hello,

I am trying to connect AnsibleForms with an AWX (on another host), but I am having issues when updating.

The error is as follows:

Failed to update awx, Error: Unknown column 'use_credentials' in 'field list'

This happens with both use_credentials and Token.

The test function does give me a correct response.

Thanks in advance.

**Comment by ansibleguy76** [comment, marked as answer]
this was fixed in 5.0.4. 
what version are you running?

**Comment by mlesierra** [comment]
I was using 5.0.3, I updated and the problem was indeed fixed, thank you very much!

**Comment by ansibleguy76** [comment]
Stale discussion... I assume this is resolved
